stests/generators/wg_110/phase_2/step_2.py

```diff
@@ -1,28 +1,40 @@
+import random
+import typing
+
 from stests.core.orchestration import ExecutionContext
 from stests.generators import utils
-from stests.generators.wg_110 import constants
+from stests.generators.wg_100 import constants
 
 
 
 # Step description.
-DESCRIPTION = "Refunds funds previously transferred from network faucet."
+DESCRIPTION = "Refunds funds previously transferred to user accounts."
 
 # Step label.
-LABEL = "refund-network-faucet"
+LABEL = "refund-users"
 
 
-def execute(ctx: ExecutionContext):
+def execute(ctx: ExecutionContext) -> typing.Callable:
     """Step entry point.
     
     :param ctx: Execution context information.
 
-    """     
-    utils.do_refund.send(
-        ctx,
-        constants.ACC_RUN_FAUCET,
-        constants.ACC_NETWORK_FAUCET,
-        True
-    )
+    """
+    # Set dispatch window.
+    deploy_count = ctx.args.user_accounts
+    dispatch_window = ctx.get_dispatch_window_ms(deploy_count)
+
+    # Refund: user -> run faucet.
+    for acc_index in range(constants.ACC_RUN_USERS, ctx.args.user_accounts + constants.ACC_RUN_USERS):
+        utils.do_refund.send_with_options(
+            args = (
+                ctx,
+                acc_index,
+                constants.ACC_RUN_FAUCET,
+                True
+            ),
+            delay=random.randint(0, dispatch_window)
+        )    
```

Comment:
These 10 lines of code is very similar to the code in stests/generators/wg_100/phase_2/step_2.py, Perhaps it could be moved into a separate method and called from those places?

Reply:
Yes was thinking of doing this refactor next.